docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,503 @@
+
+### What are your most important criteria for choosing a UI library?
+
+<img src="/static/blog/2021-survey/5.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Ranked list: 1. The design - look and feel; 2. Customizability; 3. Documentation quality; 4. Comprehensiveness; 5. Performance; 6. Popularity; 7. Accessibility; 8. Offered support and help; 9. Bundle size" />
+
+<p class="blog-description">1500 out of 1589 answered - Check <a href="https://www.typeform.com/help/a/ranking-question-360052767651/">Typeform's documentation</a> to understand ranking average calculations</p>
+
+This year we experimented with using Typeform to run the survey, which allows respondents to rank their choices.
+This could've given us a richer perspective than we've been able to capture in the past, but in the end the results weren't too different from last year.
+
+Design (look and feel) remains the most important criterion for choosing UI libraries, and customization and documentation quality are still there in the top 3.
+The notable difference this year seems to be performance climbing its way up, as it has been ranked as the fifth most important aspect.
+
+### What else can we do to improve MUI for you?
+
+<img src="/static/blog/2021-survey/6.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Word cloud of what we can do to improve MUI" />
+
+<p class="blog-description">1007 out of 1589 answered this question in 2021</p>
+
+Here are the most recurring topics for improvement:
+
+- **More components:** We continue to see requests for more advanced components such as charts, forms, and calendars, to name a few.
+- **More examples:** Many of the existing learning resources out there are now outdated following the v5 upgrade, so we have a lot of work to do to show you how to make the most of our products.
+- **Provide more themes:** Even with the launch of Material v3, Material Design is seen as outdated by many.
+  That's why we're working on a second design system that will offer the same high standard of quality as Material, but with an alternative design direction.
+- **Fewer breaking changes:** MUI Core v5 definitely introduced some important breaking changes, especially because of the new styling solution.
+  Rest assured that we don’t expect to release any major updates this year—in fact, we aim to keep majors at least 12 months apart from each other.
+- **Improve customization:** Common requests include making customization easier, providing more examples of common use-cases (like font-family and primary/secondary colors), and improving to theme capabilities.
+  Even with the popularity of Emotion and styled-components, there continues to be an enormous need to facilitate customization.
+
```

Comment:
I would propose that on top of the recurring topics for improvement (the top existing pains) we also mention the growing pains and the decreasing pain. I did a first attempt at identifying these trends, it could be pushed deeper.